docs/source/troubleshooting/lvrt-crash-dumps.rst

```diff
@@ -5,195 +5,169 @@ LabVIEW Core Dumps on Linux Real-Time
-  
+At times while troubleshooting LabVIEW Real-Time crashes, NI Support may request a memory or core dump from the system at the time of the crash. These crash dumps may provide further insight into the type of crash or where the crash is occurring, as well as information about the system. However, it's important to keep in mind that crash dumps are not guaranteed to solve an issue or even provide useful insight. They are another troubleshooting tool which can be useful in narrowing down a problem or understanding things further.
 
-Generating a Core Dump
-========================
-
-Configuring a Target for Core Dumps
------------------------------------
+The more context given with a crash dump, the more likely it is to have useful information. This can include:
 
-| By default, NI Linux Real-Time targets have core dumps disabled. This
-  setting is to ensure that these files - which can be quite large - do
-  no take up valuable storage space on embedded devices when not
-  needed. To enable core dumps, follow the steps below.
+- logs of what the application is doing at the time of the crash.
+- simplified reproducing cases.
+- context as to what changed when the crash started happening.
 
-#. Ensure that the target has software installed, is in run-mode, and
-   will not reboot into safe mode upon reboot. This can be done by
-   verifying the following in NI MAX:
+Without this information, the likelihood that a dump file will provide useful information decreases significantly. For example, a core dump may show an access violation due to a bad refrence in the code but it may not show what caused the reference to become bad in the first place. That information can be used to narrow things down but will not give a root cause on its own.
 
-   -  Verify that the **Status** of the target in
-      the **System Settings** tab is currently **Connected - Running.**
-   -  Verify that the **Force Safe Mode** checkbox is **not** checked in
-      the **System Settings** tab.
+Additionally, reproducing a crash internally to NI is generally preferred. A dump can be unwound externally, but without access to NI symbols, it is unlikely to yield useful information. Additionally, if the issue is reproduced internally at NI, the same system can be used for both reproducing and unwinding the core dump ensuring that the software versions used for debugging the dump file are consistent.
 
-#. On the **System Settings** tab in NI MAX, check the boxes
-   under **Startup Settings** for both **Enable Secure Shell Server
-   (sshd)** and **Enable Console Out.**
 
-#. Enable core dumps via one of the following methods:
+Generating a Core Dump
+========================
 
-   #. Run the following command via a terminal (serial or SSH) on the target: 
 
-      .. code:: bash
+Configuring a Target for Core Dumps
+-----------------------------------
 
-         sed -e 's|^[#[:space:]]*ulimit [[:space:]]*-c [[:space:]]*.*$|ulimit -c unlimited|' -i /etc/init.d/lvrt-wrapper
```

Comment:
I would prefer we keep the documentation for "older versions" as well for now as support is still probably going to want to use this document for older versions for quite a while.

Reply:
Agreed. I'll add the old workflow back with a note that it only applies to older LVRT configurations.